examples/newrelic/README.md

```diff
@@ -0,0 +1,193 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/examples/newrelic/README.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+## NewRelic Server Monitor Example
+
+This example shows how to run a NewRelic server monitoring daemon as a pod in a DaemonSet on an existing Kubernetes cluster.
+
+This example will create a DaemonSet which places the NewRelic monitoring daemon on every node in the cluster. It's also fairly trivial to exclude specific Kubernetes nodes from the DaemonSet to just monitor specific servers.
+
+### Step 0: Prerequisites
+
+This process will create priviliged containers which have full access to the host system for logging. Beware of the security implications of this.
```

Comment:
Isn't this the default after https://github.com/kubernetes/kubernetes/pull/13930? Would it be enough to say, "your cluster needs to allow the creation of privileged pods"? It seems like a bit of a rabbit hole to go into cluster setup in a user guide. Isn't "you should have the daemonset enabled" another similar prereq? That's even more important since it is not the default. I think you should mentions these prereqs but not explain how to implement them here.